src/wallet/scriptpubkeyman.cpp

```diff
@@ -1878,9 +1878,34 @@ int64_t DescriptorScriptPubKeyMan::GetTimeFirstKey() const
     return m_wallet_descriptor.creation_time;
 }
 
+std::unique_ptr<FlatSigningProvider> DescriptorScriptPubKeyMan::GetSigningProvider(const CScript& script, bool include_private) const
```

Comment:
This commit message in 50c3b4eee31e095a5080fcb4109cb5d8f7bfbbe7 speaks of `GetSolvingProvider`

Reply:
And? We modify `GetSolvingProvider` below. This is an internal function.